framework/pym/play/commands/junit.py

```diff
@@ -70,25 +68,19 @@ def junit(app, args):
     # Run app
     test_result = os.path.join(app.path, 'test-result')
     if os.path.exists(test_result):
+        print("~ cleaning test results")
         shutil.rmtree(test_result)
     args.append('-Dplay.autotest')
     java_cmd = app.java_cmd(args, className='play.test.Runner')
     try:
-        process = subprocess.Popen (java_cmd, env=os.environ)
+        process = subprocess.Popen(java_cmd, env=os.environ)
         signal.signal(signal.SIGTERM, handle_sigterm)
         signal.signal(signal.SIGINT, handle_sigint)
         return_code = process.wait()
         if 0 != return_code:
+            print("~ java process exited with exit code %d" % return_code)
             sys.exit(return_code)
     except OSError:
         print("Could not execute the java executable, please make sure the JAVA_HOME environment variable is set properly (the java executable should reside at JAVA_HOME/bin/java). ")
         sys.exit(-1)
 
-    if os.path.exists(os.path.join(app.path, 'test-result/result.passed')):
-        print("~ All tests passed")
-        print("~")
-        testspassed = True
-    if os.path.exists(os.path.join(app.path, 'test-result/result.failed')):
-        print("~ Some tests have failed. See file://%s for results" % test_result)
-        print("~")
-        sys.exit(1)
```

Comment:
this particular files are not generated by `play.test.Runner`

Reply:
I'm not sure what this means. Isn't it needed?

Reply:
I was confused by this code and the fact that neither `test-result/result.passed` nor `test-result/result.failed` is generated so i decided to clean it up, it doesn't look like these log messages are picked up by CI and there are another log messages about the results of the run from the java code